Leaf API.postman_collection.json

```diff
@@ -1834,7 +1835,7 @@
 										],
 										"body": {
 											"mode": "raw",
-											"raw": "{\n    \"operationsImageCreation\": true,\n    \"operationsSummaryCreation\": true,\n    \"fieldsAutoSync\" : true\n}"
+											"raw": "{\n    \"operationsImageCreation\": true,\n    \"fieldsAutoSync\" : true\n}"
```

Comment:
Should we include all the properties here?

Reply:
This is related to the PATCH, so I don't think it's obligatory to include all properties, but there's certainly no harm on putting it there. Done in 1005b0350abaa5483615d613b4390ecc3662bd8c